pymilo/transporters/cfnode_transporter.py

```diff
@@ -23,7 +23,7 @@ def __init__(self):
 
     def reset(self):
         """
-        Reset the CFNodeTransporter internal .
+        Override the father's reset function and reset the CFNodeTransporter internal data structures.
```

Comment:
I think that's too specific, can't we just keep this method's doc-string?

Reply:
applied.